mm2src/coins/lightning/ln_events.rs

```diff
@@ -284,6 +428,50 @@ impl LightningEventHandler {
                 return;
             },
         };
-        platform_coin.broadcast_transaction(&spending_tx);
+
+        self.platform.broadcast_transaction(&spending_tx);
+
+        let claiming_tx_inputs_value = outputs.iter().fold(0, |sum, output| match output {
+            SpendableOutputDescriptor::StaticOutput { output, .. } => sum + output.value,
+            SpendableOutputDescriptor::DelayedPaymentOutput(descriptor) => sum + descriptor.output.value,
+            SpendableOutputDescriptor::StaticPaymentOutput(descriptor) => sum + descriptor.output.value,
+        });
+        let claiming_tx_outputs_value = spending_tx.output.iter().fold(0, |sum, txout| sum + txout.value);
+        if claiming_tx_inputs_value < claiming_tx_outputs_value {
+            log::error!(
+                "Claiming transaction input value {} can't be less than outputs value {}!",
```

Comment:
Well, `spend_spendable_outputs` https://github.com/KomodoPlatform/atomicDEX-API/blob/6d717fc0242f61a29ad9198b2c8725c446deb278/mm2src/coins/lightning/ln_events.rs#L416 will never generate such transaction, but I added this just in case to avoid overflow in substracting if it ever happens for whatever reason.https://github.com/KomodoPlatform/atomicDEX-API/blob/6d717fc0242f61a29ad9198b2c8725c446deb278/mm2src/coins/lightning/ln_events.rs#L446

> Should we broadcast such transaction above?

Such transaction will not be accepted if broadcasted anyways, but I guess moving the transaction broadcasting below the overflow check is the better solution.